composer.json

```diff
@@ -38,19 +38,19 @@
         "phpstan/phpstan-phpunit": "^1.0",
         "phpstan/phpstan-strict-rules": "^1.0",
         "phpunit/phpunit": "^9.5.13",
-        "psalm/plugin-phpunit": "^0.16",
+        "psalm/plugin-phpunit": "^0.18",
         "rector/rector": "^0.15",
```

Comment:
rector 0.16 is released by the way

Reply:
I don't see it: https://github.com/rectorphp/rector/tags